app/views/inventories/_inventory.html.erb

```diff
@@ -0,0 +1,15 @@
+<tr>
+  <td><%= inventory.ingredient.id %></td>
+  <td><%= inventory.ingredient.name %></td>
+  <td><%= inventory.quantity %></td>
+  <td>
+    <div class= "d-flex">
+      <%= link_to 'Edit', edit_branch_inventory_path(inventory.branch_id, inventory), class: 'btn btn-primary btn-sm me-2' %>
+      <%= button_to 'Delete', branch_inventory_path(inventory.branch_id, inventory), 
+        method: :delete, 
+        class: 'btn btn-danger btn-sm', 
+        data: { confirm: 'Are you sure you want to delete this branch?'} 
```

Comment:
this is not working but fix the record class
`branch` -> `inventory`

Reply:
Done